main.py

```diff
@@ -0,0 +1,21 @@
+# This is a sample Python script.
+
+# Press Shift+F10 to execute it or replace it with your code.
+# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
+
+
+def get_recipe_price(prices, optionals=[], **ingredients):
+    price = 0
+    for ingredient in ingredients:
+        if ingredient not in optionals:
+            price += ((ingredients[ingredient] / 100) * prices[ingredient])

```

Comment:
We can try to simplify the code here -
How about to define a function which will calcuate line 11?
Also, after doing that, maybe we can simplify the for loop as commented in the other exercises.

Reply:
Yes I did it right now.